packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -43,11 +58,18 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
 
     this._id = getId('checkbox-');
     this.state = {
-      isFocused: false,
       isChecked: !!(props.checked !== undefined ? props.checked : props.defaultChecked)
     };
   }
 
+  public componentWillReceiveProps(newProps: ICheckboxProps) {
```

Comment:
why is this logic in componentWillReceiveProps()? shouldn't it be in the constructor and run once?